src/snyk/base/modules/snykLib.ts

```diff
@@ -37,11 +36,12 @@ export default class SnykLib extends BaseSnykModule implements ISnykLib {
 
       await this.contextService.setContext(SNYK_CONTEXT.FEATURES_SELECTED, true);
 
+      await this.user.identify(this.snykApiClient, this.analytics);
+      await this.contextService.setContext(SNYK_CONTEXT.LOGGEDIN, true);
```

Comment:
Is there any need to move `await this.contextService.setContext(SNYK_CONTEXT.LOGGEDIN, true);` to this line?

Reply:
ah yes, I think this was a hangover from the previous implementation. We don't need to make any changes within `SnykLib` class.